avro-fastserde/src/test/java/com/linkedin/avro/fastserde/FastGenericDeserializerGeneratorTest.java

```diff
@@ -468,7 +470,16 @@ public void shouldSkipRemovedField(Implementation implementation) {
 
     // then
     Assert.assertEquals(new Utf8("abc"), record.get("testNotRemoved"));
-    Assert.assertNull(record.get("testRemoved"));
+    if (AvroCompatibilityHelper.getRuntimeAvroVersion().laterThan(AvroVersion.AVRO_1_9)) {
+      try {
+        record.get("testRemoved");
```

Comment:
this relies on avro 1.10+ behavious that throws for fields that dont exist. would it be possibel to have a utility "assertFieldDefined" method in test util ?

Reply:
I'm not sure I get the intent of your comment.

Are you saying you'd like this same functionality moved to a utility function (but still be implemented via a try-catch)?

Or are you saying you'd like some other way of detecting if a field is missing (i.e. via reflection or by inspecting the record's schema)?

Reply:
option 2. i'd like some Helper.fieldExists(Schema, str) method

Reply:
@radai-rosenblatt what's the goal of having this new utility function, which would be used purely by the test code?